boltdesignsystem/bolt
# DS-530: Deprecate Button-Group

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-530](https://pegadigitalit.atlassian.net/browse/DS-530)

## Summary

Deprecate Button-Group and replace all uses with List

## Details

The Button-Group seemed to be a vestigial component, so we're replacing all the usage of it with Bolt-List and deprecating Button-Group

## How to test

Check out the documentation for Button-Group to make sure it was deprecated properly, then make sure the other instances have been replaced correctly.

## Release notes

Deprecate Bolt-Button-Group

### Visual changes

The buttons have also been replaced so expect the same visual changes, namely bigger text on small buttons.
